src/main/steps/switch-to-sole-application/get.ts

```diff
@@ -0,0 +1,9 @@
+import { GetController } from '../../app/controller/GetController';
+
+import { generateContent } from './content';
+
+export class SwitchToSoleApplicationGetController extends GetController {
```

Comment:
why is this needed?

Reply:
i had to remove this from app1 sequence as app2 also uses the page, i guess i could have just used a normal get controller but i noticed every other page outside of the applicant routes were uses custom controllers that did the same thing.